### PR TITLE
Fixes e2e Readme link

### PR DESCRIPTION
**Pull request by SonicZentropy**
Tiny housekeeping fix I ran into

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Overview
-This crate runs all the end-to-end tests for shuttle. These tests must run against a local dev environment, so you first have to set that up by following [these instructions](../README.md#working-on-shuttle).
+This crate runs all the end-to-end tests for shuttle. These tests must run against a local dev environment, so you first have to set that up by following [these instructions](https://github.com/shuttle-hq/shuttle/blob/main/CONTRIBUTING.md).
```
Comment (chesedo) [inline e2e/README.md:2]: Any reason for linking to `main`? I'm a bit worried about having `rc`s and stuff in the future that might have different instructions for a temporary period.

**Review by chesedo — APPROVED** [review]
Thanks for the fixup @SonicZentropy. I'm happy merging this as is, just want to check something.

File: e2e/README.md (hunk 1 above)
Comment (SonicZentropy) [reply e2e/README.md:2]: I copied it straight from the base README.  I see what you mean, so I think perhaps we should change both of them to local?

Comment (chesedo) [reply e2e/README.md:2]: Yes, if the change makes sense, then let's change both :+1:

Comment (SonicZentropy) [reply e2e/README.md]: Fixed!

**Review by chesedo — APPROVED** [review]
Thanks for the cleanup @SonicZentropy

**Pull request merged**